api/admin/routes.py

```diff
@@ -512,14 +704,25 @@ def genres():
     """Returns a JSON representation of complete genre tree."""
     return app.manager.admin_feed_controller.genres()
 
+
 @library_route('/admin/bulk_circulation_events')
 @allows_cors(allowed_domain_type=set({"admin"}))
 @returns_problem_detail
 @allows_library
 @requires_admin
 def bulk_circulation_events():
-    """Returns a CSV representation of all circulation events with optional
-    start and end times."""
+    """Return a CSV representation of all circulation events with optional start and end times.
+    ---
+    get:
+      tags:
+        - administration
+      description: Return CSV of circulation events
+      responses:
+        200:
+          description: CSV of circulation events
+          schema: BulkCirculation
+          example: [time,	event, identifier, identifier_type, title, author, fiction, audience, publisher, imprint, language, target_age, genres, location]
```

Comment:
These two attributes are not currently showing up in the API docs, but I'm also not sure how best to represent a CSV return type here. Maybe as a markdown table?